docs/ref/models/querysets.txt

```diff
@@ -3188,7 +3188,10 @@ As a convenience when no lookup type is provided (like in
 ~~~~~~~~~
 
 Exact match. If the value provided for comparison is ``None``, it will be
-interpreted as an SQL ``NULL`` (see :lookup:`isnull` for more details).
+interpreted as an SQL ``NULL`` (see :lookup:`isnull` for more details). An
+exception is :lookup:`Key and index lookups <jsonfield.key>` on
```

Comment:
"is" doesn't sound like it agrees with "lookups" to my ear -- maybe "key and index lookups are exceptions" or "behave differently"?